docs/db-driver.md

```diff
@@ -11,8 +11,8 @@ Although these methods have different names and return different stores, their s
 
 ```go
 type Driver interface {
-    NewBindingStore(string, Config) (BindingStore, error)
-    NewAuditInfoStore(string, Config) (AuditInfoStore, error)
+    NewBindingStore(name PersistenceName, params ...string) (BindingStore, error)
```

Comment:
what's params?

Reply:
Params is what will be added as a prefix for this table.

For fsc, you usually use `default`, apart from the case of the vault where you also have one per channel.

For the case of fts, you have one table per tms, so these params would be network, channel, namespace. This makes it more applicable for any application that builds on top to keep the same signature format for the store instantiation.